src/graphql/operations/networks.ts

```diff
@@ -22,8 +23,23 @@ export default function () {
     return acc;
   }, {});
 
-  return Object.entries(networks).map(([id, spacesCount]) => ({
-    id,
-    spacesCount
-  }));
+  const results = Object.entries(networks).reduce((acc, [id, spacesCount]) => {
+    acc[id] = {
+      id,
+      spacesCount: spacesCount,
+      premium: false
+    };
+
+    return acc;
+  }, {});
+
+  const premiumNetworks = await db.queryAsync(
+    'SELECT * FROM networks WHERE premium = 1'
+  );
+  premiumNetworks.forEach((network: any) => {
+    results[network.id] ||= { id: network.id, spacesCount: 0 };
+    results[network.id].premium = true;
+  });
```

Comment:
This will display a network without spaces on Explore page, we wanted to display only networks with spaces right?

Reply:
Usually new premium networks will have `0` spacesCount

Reply:
hmm sorry i missed it